### DISCUSSION TITLE
System just restarts randomly

### DISCUSSION THREAD

**Discussion by DrJeff** [tagyoureit/nodejs-poolController#498] [Q&A]
Not sure if this is an overheat issue or just a njsPC thing, as far as I see the Pi doesn't restart just njsPC. At the end of the replay is the restart.
[replayRestartIssue01.zip](https://github.com/tagyoureit/nodejs-poolController/files/8960879/replayRestartIssue01.zip)

**Comment by tagyoureit** [comment]
I looks like the process was just killed - nothing closing down njsPC.  Probably a temp issue.
I had a reheat issue on another RPI.  I validated it by outputting the CPU temp (from REM) to MQTT or another output.  It was pretty clear the Pi was overheating when the temp went sky high.

**Reply by DrJeff** [reply]
I have the temp reporting it looks like at that point in time the temp was 146°F. When is the temperature an issue at what temp? Does it throttle by killing Processes?  I have an exterior enclosure vented but the ambient temp is 102°F and inside the enclosure currently it's 109° with the Pi at 138°.

**Reply by tagyoureit** [reply]
It's supposed to be able to go up to 85c/185f, but I was starting to have issues in the 140f range.  I threw one of these [sequent fan hats](https://sequentmicrosystems.com/products/raspberry-pi-fan) on and my Pi has been much more stable ever since.

If you want to monitor it, just keep checking the temp every time it reboots.  If a pattern emerges, then that's your answer for sure.
